Feature/integrate

KelsonDenton opened the pull request:
Attempted to put board class into main for player to use.

KelsonDenton commented:
The validation still has a few bugs but my PyCharm is having trouble importing libraries for the program so I haven't actually run it to test it. I've modified some of the import statements to try to get around it but it hasn't worked. There's also a test class to test out functionality of the board class

j222144 commented:
> The validation still has a few bugs but my PyCharm is having trouble importing libraries for the program so I haven't actually run it to test it. I've modified some of the import statements to try to get around it but it hasn't worked. There's also a test class to test out functionality of the board class

have you try installing the package that is listed inside of requirement inside server folder    server --> requirement.txt in there you will see the list of package you need

shlecturer commented:
> > The validation still has a few bugs but my PyCharm is having trouble importing libraries for the program so I haven't actually run it to test it. I've modified some of the import statements to try to get around it but it hasn't worked. There's also a test class to test out functionality of the board class
> 
> have you try installing the package that is listed inside of requirement inside server folder server --> requirement.txt in there you will see the list of package you need

I added a `requirements.txt` to the reference client too. You could copy that into your own client folder.

shlecturer commented:
It would be nice to not change the reference implementation unless it really improves it. It shouldn't be changed to accommodate for your own client. :)

KelsonDenton commented:
Ooh right. So just create our own folder where we do all our own implementation in the clients directory?

KelsonDenton commented:
> > > The validation still has a few bugs but my PyCharm is having trouble importing libraries for the program so I haven't actually run it to test it. I've modified some of the import statements to try to get around it but it hasn't worked. There's also a test class to test out functionality of the board class
> > 
> > 
> > have you try installing the package that is listed inside of requirement inside server folder server --> requirement.txt in there you will see the list of package you need
> 
> I added a `requirements.txt` to the reference client too. You could copy that into your own client folder.

Oh cool thanks I added that and installed all the packages. For some reason it won't recognize the log.py file to be imported into other files though. I tried renaming the path but that didn't really help.

Sandyman commented:
> 
> 
> Ooh right. So just create our own folder where we do all our own implementation in the clients directory?

Yes. 👍
